msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -168,9 +168,9 @@ For in-meeting side panel experience only: 
      ] 
 ```
 
-### Advanced Tab SDK APIs
+### Advanced tab APIs
 
-The Microsoft Teams JavaScript client SDK is a rich SDK used to create Tabs using JavaScript. Use the latest TeamsJS (V.2.0 or later) to work in Teams, Office, and Outlook. For more information, see [Teams JavaScript client SDK](/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?tabs=javascript%2Cmanifest-teams-toolkit).
+TeamsJS is a rich library used to create Tabs using JavaScript. Use the latest TeamsJS (V.2.0 or later) to work in Teams, Office, and Outlook. For more information, see [Teams JavaScript client SDK](/microsoftteams/platform/tabs/how-to/using-teams-client-sdk).
```

Comment:
As per the update in TOC.yml line 198 and 199, please check if the link text and link also be replaced as follows:

[Teams JavaScript client library](/microsoftteams/platform/tabs/how-to/using-teams-client-library)

Reply:
Good catch! Fixed now.